front-end/package.json

```diff
@@ -4,14 +4,32 @@
   "private": true,
   "proxy": "http://backend:5000",
   "dependencies": {
+    "bootstrap": "^3.3.7",
+    "lodash": "^4.17.11",
+    "moment": "^2.22.2",
+    "node-sass-chokidar": "^1.3.3",
+    "npm-run-all": "^4.1.3",
     "react": "^16.5.0",
+    "react-bootstrap": "^0.32.4",
```

Comment:
you might not need react-bootstrap. My experience is that JS + css was still more flexible than these things. But it is up to you.